### ISSUE TITLE
Suggestion: babel/require-jsdoc (built-in require-jsdoc + @decorator not working)

### ISSUE THREAD

**Issue by cpixl** [babel/eslint-plugin-babel#48]
It seem's that the built-in rule for mandatory JSDocs ([require-jsdoc](http://eslint.org/docs/rules/require-jsdoc.html)) in classes is not accepting this:

``` javascript
/**
 * Class description
 */
@Bar
class Foo {

}
```

Only the following pass the validation:

``` javascript
@Bar
/**
 * Class description
 */
class Foo {

}
```

It would be nice if there was an override for require-jsdoc on this plugin to handle both situations :)

**Comment by Gvozd** [comment]
Please like for https://github.com/eslint/eslint/pull/8123 )

**Comment by Gvozd** [comment]
My PR to eslint rejected
Please vote for #122 )

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [@babel/eslint-plugin has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
